src/file/file.rpc.ts

```diff
@@ -75,111 +97,73 @@ export class FileServer {
     const ownerID: string = call.request.ownerID;
     const parent: string = call.request.parent;
     const size: number = parseInt(call.request.size, 10);
-    FileService.createUpload(
-      key,
-      bucket,
-      name,
-      ownerID,
-      parent,
-      size)
-      .then((upload) => {
-        callback(null, upload);
-      }).catch(err => callback(err));
+    return await FileService.createUpload(key, bucket, name, ownerID, parent, size);
```

Comment:
`await`ing a promise just to return its resolved value as a promise is useless, just return the promise.